gtrack/Eagle/README.md

```diff
@@ -0,0 +1,2 @@
+This folder contains schematic, PCB and gerber files for GSM security board.
```

Comment:
Could you add some description how any person can use these files?
For example I want to view or change them?